samri/optimization/registration.py

```diff
@@ -96,8 +97,8 @@ def structural_per_participant_test(participant,
 			res = struct_registration.run()
 
 def structural_rigid_affine(template="/Users/marksm/GitHub/mriPipeline/ants_test/template4.nii.gz",
```

Comment:
I changed it accordingly. The function name is now structural_rigid_flirt_nonlin_syn, with added docstring. I've been playing with the parameters a little. Now it works really well for me. 

Could you try on your data?